src/reporter.cc

```diff
@@ -13,7 +13,6 @@
 // limitations under the License.
 
 #include "benchmark/reporter.h"
-#include "minimal_leastsq.h"
```

Comment:
Yes, that's right.  "complexity.h" is included via "benchmark_api.h" (because of the BigO enum definition) and then via "reporter.h". As I found that the header was already included via the previous path, I just removed if from the reporter.cc. But, if you think that it will increase readability, I could add it again.